@here/harp-mapview/lib/MapView.ts

```diff
@@ -1763,8 +1763,10 @@ export class MapView extends THREE.EventDispatcher {
         this.m_tempQuat.setFromAxisAngle(EYE_INVERSE, -this.camera.rotation.z);
 
         // "any" used to fix missing .angleTo() ts error
-        const cameraAngle = (this.camera.quaternion as any).angleTo(this.m_tempQuat);
-
+        const cameraAngle = Math.min(
```

Comment:
You can combine the two min equations by clamping the camera angle to 60 degrees (i.e. acos(0.5))